backend/fms_core/template_importer/row_handlers/sample_pooling/pool.py

```diff
@@ -0,0 +1,62 @@
+
+from fms_core.template_importer.row_handlers._generic import GenericRowHandler
+
+from fms_core.services.process import create_process
+from fms_core.services.sample import pool_samples
+from fms_core.services.container import get_container, get_or_create_container
+
+class PoolsRowHandler(GenericRowHandler):
+    def __init__(self):
+        super().__init__()
+
+    def process_row_inner(self, protocol, imported_template, samples_info, pool, pooling_date, comment):
+        # Ensure there is samples_tied to the pool
+        if samples_info is None:
+            self.errors["source_sample"].append(f"Cannot find samples for pool {pool['name']}. "
```

Comment:
In addition, this is a different row handler so self.errors is an empty object and self.errors["source_sample"] is undefined (not an empty list) so you won't be able to append to it and it will crash.